src/acceptanceTest/java/uk/gov/hmcts/futurehearings/hmi/acceptance/directlisting/DirectListingValidationTest.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.futurehearings.hmi.acceptance.directlisting;
+
+import io.restassured.RestAssured;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.BeforeAll;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.futurehearings.hmi.Application;
+import uk.gov.hmcts.futurehearings.hmi.acceptance.common.test.HMICommonHeaderTest;
+
+import static io.restassured.config.EncoderConfig.encoderConfig;
+import static uk.gov.hmcts.futurehearings.hmi.acceptance.common.security.OAuthTokenGenerator.generateOAuthToken;
+
+@Slf4j
```

Comment:
Add this TestInstance(TestInstance.Lifecycle.PER_CLASS) to class to fix the error "'initialiseValues()' has incorrect signature for a @org.junit.jupiter.api.BeforeAll method "